### PR TITLE
Jshinterrorfix

### PR DESCRIPTION
**Pull request by MaanasaPriyaa**
Cleaned up code and fixed styling issues

### CODE REVIEW

File: ReportCard.js [hunk 1]
```diff
@@ -155,86 +155,113 @@ ReportCard.prototype = {
...
-   //TODO - jsdoc
-   _updateGraspCards: function(card_id, report_id, callback){
+  /**
+   * Update the card status as Received and add ReportId to the entry in grasp_cards table
+   * Call the _insertLogTbl with card_id once query is successful
+   *
+   * Function to update card status
+   * @param  {string} created_at    ISO8601 format date string
+   * @param  {string} card_id       Unique Card Id
+   * @param  {string} location      Geo coordinates in WKT format (long lat)
```
Comment (abrahamq) [inline ReportCard.js]: Awesome, thanks for the documentation.

File: ReportCard.js [hunk 2]
```diff
@@ -267,92 +294,67 @@ ReportCard.prototype = {
...
-         callback(error, null);
-       }
-       else {
-         self.logger.info('getAllReports call successful; returning results');
-         callback(error, result);
-       }
-     });
-   },
+ 						  " As l) " +
+ 					    ") As properties " +
```
Comment (abrahamq) [inline ReportCard.js]: The spacing on this still seems off?

**Comment by abrahamq** [comment]
Awesome, thanks for the fixes. I'm not super sure that we should be extending $ for put. What are your thoughts on that?

File: ReportCard.js (hunk 1 above)
Comment (MaanasaPriyaa) [inline ReportCard.js]: Of course.

**Pull request merged**